### ISSUE TITLE
Sound file not found

### ISSUE THREAD

**Issue by homosapien97** [yvt/openspades#430]
I am running Ubuntu 16.04
I have compiled and run openspades successfully before, but I just recently tried to boot it and got the following crash after moving my mouse in the server list screen:

```
2016/11/09 19:31:24 [Runner.cpp:54] [!] Unhandled exception in SDLRunner:
File not found: Sounds/Feedback/Limbo/Hover.wav
at /home/USER_NAME/openspades/openspades/Sources/Core/FileManager.cpp:49
static spades::IStream* spades::FileManager::OpenForReading(const char*) at FileManager.cpp:32
spades::audio::ALAudioChunk* spades::audio::ALDevice::CreateChunk(const char*) at ALDevice.cpp:780
virtual spades::client::IAudioChunk* spades::audio::ALDevice::RegisterSound(const char*) at ALDevice.cpp:804
void spades::ScriptContextUtils::ExecuteChecked() at ScriptManager.cpp:338
void spades::ScriptContextHandle::ExecuteChecked() at ScriptManager.cpp:277
virtual void spades::gui::MainScreen::MouseEvent(float, float) at MainScreen.cpp:81
void spades::gui::SDLRunner::Run(int, int) at SDLRunner.cpp:416
void spades::gui::Runner::Run() at Runner.cpp:75
void spades::gui::Runner::RunProtected() at Runner.cpp:48
int main(int, char**) at Main.cpp:390
 @ [/Gui/UIFramework.as:330,6] void spades::ui::UIManager::PlaySound(string)
 > [/Gui/UIControls.as:80,5] void spades::ui::ButtonBase::PlayMouseEnterSound()
 > [/Gui/UIControls.as:152,6] void spades::ui::ButtonBase::MouseEnter()
 > [/Gui/UIFramework.as:136,7] void spades::ui::UIManager::MouseEventDone()
 > [/Gui/UIFramework.as:154,5] void spades::ui::UIManager::MouseEvent(float, float)
 > [/Gui/MainScreen.as:73,4] void spades::MainScreenUI::MouseEvent(float, float)
at /home/USER_NAME/openspades/openspades/Sources/ScriptBindings/ScriptManager.cpp:361
void spades::ScriptContextUtils::ExecuteChecked() at ScriptManager.cpp:338
void spades::ScriptContextHandle::ExecuteChecked() at ScriptManager.cpp:277
virtual void spades::gui::MainScreen::MouseEvent(float, float) at MainScreen.cpp:81
void spades::gui::SDLRunner::Run(int, int) at SDLRunner.cpp:416
void spades::gui::Runner::Run() at Runner.cpp:75
void spades::gui::Runner::RunProtected() at Runner.cpp:48
int main(int, char**) at Main.cpp:390

2016/11/09 19:33:26 [Settings.cpp:268] Saving preference to SPConfig.cfg
2016/11/09 19:33:26 [Settings.cpp:184] Saving preferences to SPConfig.cfg
AL lib: (EE) alc_cleanup: 1 device not closed
```

I tried running with audio disabled, and so it didn't crash in the server menu. I could select a server and try to connect to it. However, when I connected, I got this error:

```
2016/11/09 19:39:50 [MainScreen.cpp:256] [!] Error while running a game client: File not found: Textures/Smoke1/000.png
at /home/USER_NAME/openspades/openspades/Sources/Core/FileManager.cpp:49
static spades::IStream* spades::FileManager::OpenForReading(const char*) at FileManager.cpp:32
spades::draw::GLImage* spades::draw::GLImageManager::CreateImage(const string&) at GLImageManager.cpp:73
spades::draw::GLImage* spades::draw::GLImageManager::RegisterImage(const string&) at GLImageManager.cpp:51
virtual spades::client::IImage* spades::draw::GLRenderer::RegisterImage(const char*) at GLRenderer.cpp:263
virtual void spades::client::Client::RunFrame(float) at Client.cpp:399
virtual void spades::gui::MainScreen::RunFrame(float) at MainScreen.cpp:244
void spades::gui::SDLRunner::Run(int, int) at SDLRunner.cpp:416
void spades::gui::Runner::Run() at Runner.cpp:75
void spades::gui::Runner::RunProtected() at Runner.cpp:48
int main(int, char**) at Main.cpp:390

Wed Nov  9 19:39:50 2016 Disconnecting
2016/11/09 19:39:50 [Client.cpp:227] Disconnected
2016/11/09 19:39:50 [GLRenderer.cpp:307] New map loaded; freeing old renderers...
2016/11/09 19:39:50 [GLRenderer.cpp:346] No map loaded
2016/11/09 19:39:50 [GLRenderer.cpp:307] New map loaded; freeing old renderers...
2016/11/09 19:39:50 [GLRenderer.cpp:322] Creating new renderers...
2016/11/09 19:39:50 [GLRenderer.cpp:324] Creating Terrain Shadow Map Renderer
2016/11/09 19:39:50 [GLRenderer.cpp:326] Creating TerrainRenderer
2016/11/09 19:39:50 [GLRenderer.cpp:328] Creating Minimap Renderer
2016/11/09 19:39:50 [GLRenderer.cpp:330] Creating Water Renderer
2016/11/09 19:39:50 [GLRenderer.cpp:339] Radiosity is disabled
2016/11/09 19:39:50 [GLRenderer.cpp:344] Created
2016/11/09 19:39:56 [SDLRunner.cpp:223] Close requested by Client
2016/11/09 19:39:56 [SDLRunner.cpp:306] Leaving Client Loop
2016/11/09 19:39:56 [Settings.cpp:268] Saving preference to SPConfig.cfg
2016/11/09 19:39:56 [Settings.cpp:184] Saving preferences to SPConfig.cfg
```

**Comment by 100best** [comment]
It is becauce DevPaks29.zip which contains non-free sounds is missing files required by openspades.  A temporary workaround would be to edit `Resources/PakLocation.txt` and change "Non GPL-compatible data pak" url to `https://www.dropbox.com/s/aprb9aufqk2abe1/DevPaks28.zip?dl=1`.

**Comment by yvt** [comment]
As of #428, some sound files previously provided by DevPaks are relocated and now included in the source tree, but I've forgot to update CMakeFiles to accommodate this change. 

Fixed in 2872e4e and should be okay now.
